Apps/WebClient/src/ClientApp/src/components/authentication/LogoutCompleteView.vue

```diff
@@ -21,13 +23,13 @@ function getRedirectTimeout() {
         return 10000;
     }
     return configValue;
-}
+});
 
-setTimeout(() => {
-    if (route.path == Path.LogoutComplete) {
-        router.push({ path: Path.Root });
-    }
-}, getRedirectTimeout());
+onUnmounted(() => clearTimeout(redirectTimeoutId.value));
+
+redirectTimeoutId.value = setTimeout(() => {
+    router.push({ path: Path.Root });
```

Comment:
```suggestion
    router.push(Path.Root);
```

didn't you want to avoid using the path property?

Reply:
I realise in this instance it likely doesn't matter.

Reply:
Right, it's more important when we do not know exactly where we are redirecting to, but I will update this one so it doesn't get copy/pasted somewhere and end up causing that sort of issue.